packages/graphback-core/src/plugin/GraphbackCoreMetadata.ts

```diff
@@ -41,9 +41,11 @@ export class GraphbackCoreMetadata {
 
     /**
      * Get Graphback Models - GraphQL Types with additional CRUD configuration
+     * 
+     * @param forceRefresh - refresh models calculated from schema
```

Comment:
This implementation is still flawed. I think we have this optimalization to get models only once, but if we will have plugin that creates model then this will not be included. Same if we add new field (for relationships etc.)